### PR TITLE
DevOps Independent 1

### PR DESCRIPTION
**Pull request by LukeP91**

### CODE REVIEW

File: content/devops/independent/iaac_advanced.md [hunk 1]
```diff
@@ -6,20 +6,29 @@ title = "IaaC Advanced"
...
-**Description:** You understand the IaaC solutions used extensivly and follow the best practices. 
+**Description:** You understand the IaaC solutions used extensivly and follow the best practices.
 
 **Person who successfully completed requirement for given block can:**
 
 - Improve and provide valuable feedback for current IaaC code.
 - Create complex modules that solve specific needs
 - Incorporate Terraform as part of continous integration process
-- Use Terraform manage different parts of project not only infrastructure
+- Use Terraform to manage different parts of project not only infrastructure
```
Comment (qbart) [inline content/devops/independent/iaac_advanced.md:18]:
```suggestion
- Use Terraform to manage different parts of project, not only infrastructure
```

File: content/devops/independent/security_advanced.md [hunk 2]
```diff
@@ -21,6 +21,23 @@ title = "Security Advanced"
...
+- 📗 [Vault documentation](https://www.vaultproject.io/docs)
+- 📗 [Vault exam review guide](https://learn.hashicorp.com/tutorials/vault/associate-review)
+
 ### 📝 Katas
+- Setup different Vault policies that allow only access to selected resources based on the role of the user
+- Implement AppRole in Vault
+- Prepare a sample threat model for a project you work on
+- Implement Trivy as part of your CI/CD pipeline
 ### 🎤 Interview
+- What are the most common infrastructure misconfigurations in AWS that can easily be prevented?
```
Comment (qbart) [inline content/devops/independent/security_advanced.md:36]:
```suggestion
- What are the most common infrastructure misconfigurations in AWS that can be easily prevented?
```

File: content/devops/independent/iaac_advanced.md [hunk 3]
```diff
@@ -6,20 +6,29 @@ title = "IaaC Advanced"
...
 
 {{% /bubble%}}
 
 ### 🎓 Learn
+- 📗 [Terraform](https://www.terraform.io/)
+- 📗 [Terraform certification review guide](https://learn.hashicorp.com/tutorials/terraform/associate-review?in=terraform/certification)
+- 📗 [Terraform providers](https://www.terraform.io/docs/providers/index.html)
+- 📗 [Terraform CLI Documentation](https://www.terraform.io/docs/cli-index.html)
 ### 📝 Katas
+- Configure Terrafrom Cloud
```
Comment (qbart) [inline content/devops/independent/iaac_advanced.md:28]: maybe sth like:
> Configure terraform to use TF Cloud?

**Comment by PoslinskiNet** [comment]
@qbart @LukeP91 is it ready to merge?

**Comment by LukeP91** [comment]
@PoslinskiNet I should finish it by the end of the week

File: content/devops/independent/security_advanced.md (hunk 2 above)
Comment (LukeP91) [reply content/devops/independent/security_advanced.md:36]: 👍

File: content/devops/independent/iaac_advanced.md (hunk 1 above)
Comment (LukeP91) [reply content/devops/independent/iaac_advanced.md:18]: 👍

**Review by qbart — APPROVED** [review]

**Pull request merged**